app/exporters/section_publishing_api_exporter.rb

```diff
@@ -113,7 +112,7 @@ def update_type
   end
 
   def rendered_document_attributes
-    @rendered_document_attributes ||= document_renderer.call(document).attributes
+    @rendered_document_attributes ||= ManualDocumentRenderer.new.call(document).attributes
```

Comment:
I think I'm happy with it for now given that I suspect it'll change in future commits.